src/debuginfo.cpp

```diff
@@ -491,6 +499,13 @@ void lookup_pointer(DIContext *context, char **name, size_t *line,
     }
     *line = info.getLine();
     jl_copy_str(filename, info.getFileName());
+
+    if (inlineinfo.getNumberOfFrames() > 1) {
+        topinfo = inlineinfo.getFrame(inlineinfo.getNumberOfFrames() - 1);
+        if (strcmp(topinfo.getFunctionName(), "<invalid>") == 0) goto done;
```

Comment:
jumping to done will delete this frame. i think it might be better to just allow "<invalid>" to show up if this is where the code actually managed to arrive at